app/app.go

```diff
@@ -123,12 +121,16 @@ func NewGaiaApp(
 		interfaceRegistry: interfaceRegistry,
 		invCheckPeriod:    invCheckPeriod,
 	}
+
+	moduleAccountAddresses := app.ModuleAccountAddrs()
+
 	// Setup keepers
 	app.AppKeepers = keepers.NewAppKeeper(
 		appCodec,
 		bApp,
 		legacyAmino,
 		maccPerms,
+		moduleAccountAddresses,
```

Comment:
Call is duplicated on [line 291](https://github.com/cosmos/gaia/pull/1902/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7L289)